src/app/charts/chart.component.html

```diff
@@ -11,6 +11,12 @@ <h2 class="chart-label">
         </button>
     </div>
 </div>
+<div *ngIf="chart.indicator.thresholdIndicator" class="chart-options">
```

Comment:
There are five varieties of components we need, or maybe just three, if the overlapping ones can be combined. Is that too many?